WeScan/Extensions/CIImage+Utils.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CIImage+Utils.swift
+//  WeScan
+//
+//  Created by Julian Schiavo on 14/11/2018.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import CoreImage
+import Foundation
+
+extension CIImage {
+    func applyingAdaptiveThreshold() -> UIImage? {
```

Comment:
Can you add documentation to this method?

Reply:
Done in c9427fd